tests/fixtures/autouse_fixtures.py

```diff
@@ -119,10 +117,7 @@ def seed_all(rank_zero_seed: int, monkeypatch: pytest.MonkeyPatch):
 @pytest.fixture(autouse=True)
 def mapi_fixture(monkeypatch):
     # Composer auto-adds mosaicml logger when running on platform. Disable logging for tests.
-    future_obj = Future()
-    future_obj.set_result(None)
-    mock_update = lambda *args, **kwargs: future_obj
-    monkeypatch.setattr(mcli, 'update_run_metadata', mock_update)
+    monkeypatch.setenv('MOSAICML_PLATFORM', 'False')
```

Comment:
can you confirm for me that the actual mosaic ml logger tests still function as expected?

Reply:
It still passes? The test looks to verify mosaicml logger logged certain items, eg
```assert 'mosaicml/training_progress' in metadata```

and this works fine